src/madhavi_Khasbage/ArrayPrograms/FindMaxNumber.java

```diff
@@ -0,0 +1,42 @@
+package madhavi_Khasbage.ArrayPrograms;
+
+import java.util.Scanner;
+/*
+Program 6:  From given array return max number.
+input : 22,35,65,88,11,23,45
+output : 88
+*/
+public class FindMaxNumber {
+
+	int findMaximumNum(int[] arr) {
+		int temp = 0;
+		int arrLength = arr.length;
+		for (int index = 0; index < arr.length; index++) {
+			for (int i = index + 1; i < arr.length; i++) {
```

Comment:
you don't require two loops, you can handle with single loop

int max = arr[0];
		for (int i = 1; i < arr.length; i++) {
			if (max < arr[i]) {
				max = arr[i];
			}
		}